src/pkg/cli/client/playground.go

```diff
@@ -116,9 +116,9 @@ func (g PlaygroundProvider) RemoteProjectName(ctx context.Context) (string, erro
 		return "", err
 	}
 	if resp.Project == "" {
-		return "", errors.New("no projects found")
+		return "", errors.New("no Playground projects found")
```

Comment:
Same thing here

Reply:
Ditto: no clue here what the user is doing, just that there's no project deployed. Do they expect one and it's not there?